bindings/hyperdrivepy/src/hyperdrive_state_methods/lp/remove.rs

```diff
@@ -102,8 +102,21 @@ impl HyperdriveState {
         let (withdrawal_shares_redeemed, share_proceeds, _state) = result;
 
         Ok((
-            U256::from(share_proceeds).to_string(),
-            U256::from(withdrawal_shares_redeemed).to_string(),
+            U256::try_from(share_proceeds)
+                .map_err(|err| {
+                    PyErr::new::<PyValueError, _>(format!(
+            "Failed to convert calculate_remove_liquidity.share_proceeds result {} to U256: {}",
```

Comment:
I actually like this syntax better than the index used above; what do you think about using it across the board?